lib/plugins/aws/customResources/resources/eventBridge/lib/utils.test.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
```

Comment:
Let's move out this file to `lib/aws/customResources/test/eventBridge.test.js`

Otherwise it'll be bundled into custom resource lambda